datadog_lambda/tracing.py

```diff
@@ -233,6 +233,18 @@ def extract_context_from_sqs_or_sns_event_or_context(event, lambda_context):
 
     Falls back to lambda context if no trace data is found in the SQS message attributes.
     """
+
```

Comment:
We should change the title of this method

Reply:
Since we are including EventBridge? or what name would be more clear?

I think the current naming is adecuate since we are extracting context from either SNS or SQS.

`SNS => SQS` and `EB => SQS` are variants of it.

If you ask me, I'd refactor the whole method to separate SNS and SQS. Due to how they change between the type and value.

Reply:
yeah probably better to refactor, but let's just rename for now.